packages/server/graphql/public/typeDefs/TeamMemberIntegrations.graphql

```diff
@@ -0,0 +1,54 @@
+"""
+All the available integrations available for this team member
+"""
+type TeamMemberIntegrations {
```

Comment:
+1 Could you please remove `type TeamMemberIntegrations` from `_legacy.graphql` now that it lives in its own home?